workspaces/landing/pages/landing/landing.components.tsx

```diff
@@ -53,63 +55,64 @@ const C = {
   ),
   Features: () => (
     <S.FeaturesGrid>
-      <div>
-        <Heading variant={500}>Dashboard</Heading>
+      <section>
```

Comment:
those divs were fine. I've thought of single section for the whole block (features grid)